Content.Server/Power/Components/ChargerComponent.cs

```diff
@@ -26,5 +31,12 @@ public sealed partial class ChargerComponent : Component
         /// </summary>
         [DataField("whitelist")]
         public EntityWhitelist? Whitelist;
+
+        /// <summary>
+        /// Indicates whether the charger is portable and thus subject to EMP effects
+        /// and bypasses checks for transform, anchored, and ApcPowerReceiverComponent.
+        /// </summary>
+        [DataField("portable")]
+        public bool Portable = false;
```

Comment:
```suggestion
        /// <summary>
        /// Indicates whether the charger is portable and thus subject to EMP effects
        /// and bypasses checks for transform, anchored, and ApcPowerReceiverComponent.
        /// </summary>
        [DataField]
        public bool Portable = false;
```

I thought about this for 10 minutes on whether it should be its own component or not but I think this is fine as it would make the GetCharge a lot more convoluted.